lib/pages/settings_page.dart

```diff
@@ -83,7 +83,7 @@ class _SettingsPageState extends State<SettingsPage> {
               textAlign: TextAlign.center,
               style: Theme.of(context).textTheme.headline1),
           SizedBox(height: 10),
-          Expanded(flex: 2, child: settingsWidget),
+          Expanded(flex: 3, child: settingsWidget),
```

Comment:
@saachipahwa Also I changed this cos the text was taking up too much space.

Reply:
No prob